### ISSUE TITLE
Command Line uses absolute paths only

### ISSUE THREAD

**Issue by noio** [spriteuv/SpriteUV2#4]
When running SpriteUV 2 from the command line, passing in a relative path, SpriteUV interprets it as an absolute path (by prefixing "/") and errors. 

This is relatively easy to work around, but makes it harder to share scripts with collaborators.

```
➜  cloud-garden-git /Applications/SpriteUV2.app/Contents/MacOS/SpriteUV2 -b ./3d-assets/Blocks/texture_atlas_settings.suv
{"author": "Ildar Valeev", "id": "a52e0d0a6f1b8733", "licenseKey": "XXXXXXXXXXXXXXXXXXXXXXXXX", "version": {"id": "1.55", "name": "SpriteUV2", "type": "Free", "ver": "f"}}
Build:
./3d-assets/Blocks/texture_atlas_settings.suv
Loading files...
ERROR: USUV file: /./3d-assets/Blocks/texture_atlas_settings.suv cannot be loaded```

**Comment by spriteuv** [comment]
Will be fixed on upcoming update

**Comment by spriteuv** [comment]
Fixed in 1.61.
